apps/avatax/src/modules/avatax/order-cancelled/avatax-order-cancelled-payload-transformer.test.ts

```diff
@@ -7,23 +7,19 @@ const configMockGenerator = new AvataxConfigMockGenerator();
 const avataxMockConfig = configMockGenerator.generateAvataxConfig();
 
 describe("AvataxOrderCancelledPayloadTransformer", () => {
-  it("throws an error when order = null", () => {
-    const payload = { order: null } as any as OrderCancelledPayload;
-    const transformer = new AvataxOrderCancelledPayloadTransformer(avataxMockConfig);
-
-    expect(() => transformer.transform(payload)).toThrow("Order is required");
-  });
   it("throws an error when no avataxId is present", () => {
     const payload = { order: {} } as any as OrderCancelledPayload;
     const transformer = new AvataxOrderCancelledPayloadTransformer(avataxMockConfig);
 
-    expect(() => transformer.transform(payload)).toThrow();
+    // @ts-expect-error
```

Comment:
**Question:** why we have expect error here?

Reply:
Because we're passing `avataxId` that doesn't exist and transformer expects a string.